src/Scope-Simulation_Driver/main.py

```diff
@@ -97,6 +100,12 @@ def get_GUIparameter(self, parameter: dict) -> None:  # noqa: N802
         self.sampling_rate_type = parameter["SamplingRate"]
         self.sampling_rate = float(parameter["SamplingRate"])
```

Comment:
`sample_rate_type` should reference the right corresponding key :
```suggestion
        self.sampling_rate_type = parameter["SamplingRateType"]
        self.sampling_rate = float(parameter["SamplingRate"])
```